system/infra-monitoring/values.yaml

```diff
@@ -176,4 +176,9 @@ vrops_exporter:
   scrapeTimeout: 295s
 
 px_exporter:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+esxi_exporter:
+  enabled: false
+  scrapeInterval: 60m
```

Comment:
is a single run taking 60m? If not, what is the expected runtime here?

Reply:
Depends on the region, but the information doesn't needs to be updated more frequently.